### DISCUSSION TITLE
SSH forwarding to multiple servers?

### DISCUSSION THREAD

**Discussion by wysegy66** [ContainerSSH/ContainerSSH#339] [Questions]
Can I use a single ContainerSSH server (E.g. running on Linux) to proxy to different SSH servers?

**Comment by ghost** [comment]
> Can I use a single ContainerSSH server (E.g. running on Linux) to proxy to different SSH servers?

Yes you can! The config webhook lets you dynamically configure which SSH server to proxy the connection to. You can find the documentation for the webhook [in the reference manual](https://containerssh.io/reference/configserver/). Please let us know if you have any further questions.
